### ISSUE TITLE
Does the WinAppDriver UI Recorder support windows server 2016?

### ISSUE THREAD

**Issue by namilantu** [Microsoft/WinAppDriver#549]

**Comment by macervera** [comment]
I'm interested in this question too.

I'm thinking installing a Terminal Server Machine with 10 users, and control parallel testing execution using Jenkins slave nodes.

Any experience in this?

**Comment by gdang** [comment]
Yes, the recorder does work on Server 2016. I have successfully recorded some UI tests on that OS inside a VM (64-bit).
